Compilation fails on Windows in mingw Ubuntu cross compile

```
/usr/bin/x86_64-w64-mingw32-ld: cannot find -lWinmm                                                                                                                                                                                                                                                                 
collect2: error: ld returned 1 exit status                                                                                                                                                                                                                                                                          
/home/cschol/src/Rack-SDK/Rack-SDK/compile.mk:52: recipe for target 'plugin.dll' failed                                                                                                                                                                                                                             
make: *** [plugin.dll] Error 1                                                                
```

Fix:

```
diff --git a/Makefile b/Makefile
index 6b2b99f..ce9a285 100644
--- a/Makefile
+++ b/Makefile
@@ -91,7 +91,7 @@ endif
 
 ifdef ARCH_WIN
 SOURCES += $(SRL)/filesystem/filesystem.cpp
-LDFLAGS += -lWinmm
+LDFLAGS += -lwinmm
 endif
 
 ifdef ARCH_LIN
```

--- a/Makefile
+++ b/Makefile
@@ -91,7 +91,7 @@ endif
 
 ifdef ARCH_WIN
 SOURCES += $(SRL)/filesystem/filesystem.cpp
-LDFLAGS += -lWinmm
+LDFLAGS += -lwinmm
 endif
 
 ifdef ARCH_LIN

